src/test/java/uk/gov/hmcts/reform/professionalapi/domain/entities/DXAddressTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.professionalapi.domain.entities;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import org.junit.Test;
+
+public class DXAddressTest extends AbstractEntityTest{
+
+	@Test
+	public void creates_dx_address_correctly() {
+
+		ContactInformation contactInformation = mock(ContactInformation.class);
+
+		DXAddress dxAddress = new DXAddress("some-number", "some-exchange", contactInformation);
+
+		assertThat(dxAddress.getDxNumber()).isEqualTo("some-number");
+		assertThat(dxAddress.getDxExchange()).isEqualTo("some-exchange");
+		assertThat(dxAddress.getContactInformation()).isEqualTo(contactInformation);
+		assertThat(dxAddress.getId()).isNull();
+	}
+

```

Comment:
We don't have a test to validate if Dx,address array provided without dxnumber and dxexchange

Reply:
We have covered this part in Integration tests.